### DISCUSSION TITLE
Core Not Found, please download it before starting.

### DISCUSSION THREAD

**Discussion by Wikevin** [UnblockNeteaseMusic/luci-app-unblockneteasemusic#127] [Q&A] [answered]
Failed to check latest core version, please try again later.
Core Not Found, please download it before starting.

**Comment by 1715173329** [comment, marked as answer]
你网烂了。

**Reply by Wikevin** [reply]
![image](https://user-images.githubusercontent.com/28334691/156159050-bccdc640-1768-4beb-b3de-eaf5a2b57e01.png)
啊这，我关掉代理之后再更新内核，更新到最新还是无法启动

**Reply by Wikevin** [reply]
![image](https://user-images.githubusercontent.com/28334691/156162686-f3bba35d-748a-47b5-b96c-3b15fb9db40c.png)
我按照下面无法更新内核的解决方式把文件都更新到最新，然后再次启动，依然显示这个问题
然后日志里面没有报错

**Reply by bwsnb** [reply]
将那五个文件 手动传到/usr/share/unblockneteasemusic/core   这个目录就可以了

**Reply by Wikevin** [reply]
试过了也不行
最后更新openwart之后才解决问题。
